spot-client/src/common/remote-control/xmpp-connection.js

```diff
@@ -579,10 +585,14 @@ export default class XmppConnection extends Emitter {
                 return;
             }
 
+            const reconnectJitter = getJitterDelay(Math.min(4, this._reconnectsAttempted), 1000, 3);
```

Comment:
I think `this._reconnectsAttempted += 1;` should be just above this line to start from 1